python/pyarrow/_fs.pyx

```diff
@@ -100,6 +100,48 @@ cdef class FileInfo(_Weakrefable):
         If given, the filesystem entry size in bytes.  This should only
         be given if `type` is `FileType.File`.
 
+    Examples
+    --------
+    Generate a file:
+
+    >>> from pyarrow import fs
+    >>> local = fs.LocalFileSystem()
+    >>> path_fi = local_path + '/pyarrow-fi-example.dat'
```

Comment:
This is "fi" meant to be "fs"?

Reply:
Nah, "fi" means to represent "FileInfo" - me trying to come up with different names for the file 😬

Reply:
Does it have to be different? I assume that `open_output_stream` can overwrite a file, and the reader doesn't know that the fixture already created such a file

Reply:
You are right. Will correct! +1